Model from 01.06.2016/Json_examples/example_msg.json

```diff
@@ -0,0 +1,10 @@
+{
+  "doctype": "msg",
+  "timestamp": "1458518400",
+  "from": "",
+  "to": "",
+  "title": "Wie gehts?",
+  "content": "Wie gehts dir? Würde mich über ein nettes Gespräch freuen.",
+  "deletedFrom": "false",
```

Comment:
JSON can handle Booleans: http://www.tutorialspoint.com/json/json_data_types.htm
That would be better than using a String here